harness/tests/determined/common/experimental/test_experiment.py

```diff
@@ -147,3 +149,39 @@ def test_list_trials_requests_pages_lazily(
         assert len(responses.calls) == page_num
     total_pages = math.ceil((len(tr_resp.trials)) / page_size)
     assert len(responses.calls) == total_pages
+
+
+@responses.activate
```

Comment:
Don't think the decorator is needed (or helpful).

Reply:
🤦